sdk/go/common/resource/plugin/provider.go

```diff
@@ -114,6 +117,10 @@ type DiffConfigRequest struct {
 type DiffConfigResponse = DiffResult
 
 type ConfigureRequest struct {
+	URN    *resource.URN
+	Name   *string
+	Type   *tokens.Type
+	ID     *resource.ID
```

Comment:
Worth bringing over the comments from the proto here?

Reply:
Yes